bitwarden_license/src/Commercial.Infrastructure.EntityFramework/Repositories/ServiceAccountRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoMapper;
+using Bit.Core.Repositories;
+using Bit.Infrastructure.EntityFramework.Models;
+using Bit.Infrastructure.EntityFramework.Repositories;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Bit.Commercial.Infrastructure.EntityFramework.Repositories
+{
+    public class ServiceAccountRepository : Repository<Core.Entities.ServiceAccount, ServiceAccount, Guid>, IServiceAccountRepository
+    {
+        public ServiceAccountRepository(IServiceScopeFactory serviceScopeFactory, IMapper mapper)
+            : base(serviceScopeFactory, mapper, db => db.ServiceAccount)
+        {
+
+        }
```

Comment:
```suggestion
        { }
```

Reply:
Went ahead and made this formatting change in Secret and Project repos as well to keep consistent. 

Resolved in https://github.com/bitwarden/server/pull/2301/commits/e2746d040e644a4550ba4d15595dabc375a8adf8